src/app/lib/utilities/ichefURL/index.js

```diff
@@ -18,9 +18,27 @@ const buildPlaceholderSrc = (src, resolution) => {
   return newUrl.join('/');
 };
 
+const buildPipsPlaceholderSrc = (src, resolution) => {
+  const parts = src.split('/');
+  const [protocol, blank, domain, type, imgService, ...extraParts] = parts;
+  if (protocol !== 'https:') return src;
+  const definedWidth = `${resolution[0]}x${resolution[1]}`;
+  extraParts.shift();
```

Comment:
The base resolution in the locator link. this gets replaced by the definedWidth.
Now that I think of it, definedWidth is the new resolution that gets substituted to the old one, so I will change the variable name to definedResolution to avoid confusion 👍